src/Markdown.ts

```diff
@@ -22,7 +22,7 @@ import { logger } from "matrix-js-sdk/src/logger";
 
 import { linkify } from "./linkify-matrix";
 
-const ALLOWED_HTML_TAGS = ["sub", "sup", "del", "u"];
+const ALLOWED_HTML_TAGS = ["sub", "sup", "del", "u", "br", "br/"];
```

Comment:
presumably `br/` is now redundant here?

Reply:
No, I'd say not. This is actually a bit broader than what we need. The tags listed here are allowed _anywhere_ in a message, not just inside user pills. So if we remove `br/`, user pills would still work but you could not use `<br/>` in a message even though you can us `<br>` which feels a bit non-intuitive to me.